[freerdp/libusb] build failure

**Host Environment**
- OS: x64-linux
- latest vcpkg master

**To Reproduce**
Steps to reproduce the behavior:
`./vcpkg install libusb:x64-linux`

**Failure logs** 
```
Building package libusb[core]:x64-linux...
--
737 | CMake Error at ports/libusb/portfile.cmake:2 (message):
738 | Error: the port is unsupported on your platform.  Please open an issue on
739 | github.com/Microsoft/vcpkg to request a fix
740 | Call Stack (most recent call first):
741 | scripts/ports.cmake:90 (include)
742 |  
743 |  
744 | Error: Building package libusb:x64-linux failed with: BUILD_FAILED
745 | Please ensure you're using the latest portfiles with `.\vcpkg update`, then
746 | submit an issue at https://github.com/Microsoft/vcpkg/issues including:
747 | Package: libusb:x64-linux
748 | Vcpkg version: 2020.02.04-unknownhash-external
749 |  
750 | Additionally, attach any relevant sections from the log files above.
751 | The command '/bin/sh -c ./vcpkg install freerdp' returned a non-zero code: 1
752 | time="2020-05-11T07:08:47Z" level=fatal msg="exit status 1"
```


**Additional context**
I'm using the freerdp port which depends on libusb. I don't even make use of libusb, so an alternate solution for my case would be to make the libusb dependency optional.

Relevant PR: https://github.com/microsoft/vcpkg/pull/11051 (added the libusb dependency)

--- a/ports/freerdp/CONTROL
+++ b/ports/freerdp/CONTROL
@@ -1,5 +1,9 @@
 Source: freerdp
-Version: 2.0.0
+Version: 2.0.0-1
 Homepage: https://github.com/FreeRDP/FreeRDP
 Description: A free implementation of the Remote Desktop Protocol (RDP)
-Build-Depends: openssl, libusb, glib (!windows)
+Build-Depends: openssl, glib (!windows)
+
+Feature: urbdrc
+Description: USB redirection channel support
+Build-Depends: libusb

--- a/ports/freerdp/portfile.cmake
+++ b/ports/freerdp/portfile.cmake
@@ -26,11 +26,16 @@ file(WRITE "${SOURCE_PATH}/.source_version" "${SOURCE_VERSION}-vcpkg")
 
 file(REMOVE ${SOURCE_PATH}/cmake/FindOpenSSL.cmake) # Remove outdated Module
 
+vcpkg_check_features(OUT_FEATURE_OPTIONS FEATURE_OPTIONS
+    urbdrc CHANNEL_URBDRC
+)
+
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
     OPTIONS
         ${FREERDP_CRT_LINKAGE}
+        ${FEATURE_OPTIONS}
 )
 
 vcpkg_install_cmake()
@@ -53,13 +58,13 @@ if(VCPKG_LIBRARY_LINKAGE STREQUAL "dynamic")
         file(COPY ${FREERDP_DLL} DESTINATION ${CURRENT_PACKAGES_DIR}/bin)
         file(REMOVE ${FREERDP_DLL})
     endforeach()
-    
+
     file(GLOB_RECURSE FREERDP_DLLS "${CURRENT_PACKAGES_DIR}/debug/lib/*.dll")
     foreach(FREERDP_DLL ${FREERDP_DLLS})
         file(COPY ${FREERDP_DLL} DESTINATION ${CURRENT_PACKAGES_DIR}/debug/bin)
         file(REMOVE ${FREERDP_DLL})
     endforeach()
-else()    
+else()
     file(GLOB_RECURSE FREERDP_TOOLS "${CURRENT_PACKAGES_DIR}/bin/*")
     foreach(FREERDP_TOOL ${FREERDP_TOOLS})
         file(COPY ${FREERDP_TOOL} DESTINATION ${CURRENT_PACKAGES_DIR}/tools/${PORT})
@@ -112,7 +117,7 @@ vcpkg_replace_string(${CURRENT_PACKAGES_DIR}/share/FreeRDP-Client/FreeRDP-Client
     "bin/freerdp-client2${VCPKG_TARGET_SHARED_LIBRARY_SUFFIX}"
 )
 
-if(VCPKG_LIBRARY_LINKAGE STREQUAL "static")
+if(VCPKG_TARGET_IS_WINDOWS AND VCPKG_LIBRARY_LINKAGE STREQUAL "static")
     file(GLOB OBJS ${CURRENT_PACKAGES_DIR}/debug/*.lib)
     file(REMOVE ${OBJS})
     file(GLOB OBJS ${CURRENT_PACKAGES_DIR}/*.lib)

